<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="SearchWidget" name="widget_search" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>471</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="DirectionWidget" name="widget_direction" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>471</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>600</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>640</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="MapWidget" name="widget_map" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchWidget</class>
   <extends>QWidget</extends>
   <header>searchwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DirectionWidget</class>
   <extends>QWidget</extends>
   <header>directionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>mapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
